site/en/blog/chips-origin-trial/index.md

```diff
@@ -131,7 +120,7 @@ Not every client will have the origin trial enabled.
 
 ### Cookies requirements
 
--   Partitioned cookies must be set with the `Secure` and `Path=/` and without the `Domain` attribute. 
+-   Partitioned cookies must be set with the `Secure` and `Path=/` and without the `Domain` attribute.
```

Comment:
Nit: the no-Domain requirement is not going to be enforced in M104, though it is TBD if this change is permanent.

Reply:
I'm gonna edit this once the change lands in 104